src/backends/torch/torchmodule.h

```diff
@@ -50,10 +50,8 @@ namespace dd
 
     void freeze_traced(bool freeze);
 
-    /** Add linear model at the end of module. Automatically detects size of
-     * the last layer thanks to the provided example output.*/
```

Comment:
It would have been great to keep the comment

Reply:
tks, done.